src/commands/config/config.type.ts

```diff
@@ -0,0 +1,12 @@
+import { DnsCreateTxtRecordCommand } from "../../commands/dns/txt-record/dns-command.type";
+import { DeployDocumentStoreCommand, DeployTokenRegistryCommand } from "../deploy/deploy.types";
+import { CreateWalletCommand } from "../wallet/wallet.type";
+
+export interface CreateConfigCommand
+  extends CreateWalletCommand,
```

Comment:
why does this extend so many command, while there are only 2 options for the create command ? (outputDir and wallet path ?)

![image](https://user-images.githubusercontent.com/503185/107472433-ed4fb100-6ba9-11eb-8580-18da7deb4a56.png)

Also it's weird to have both outputFile and outputDir from the type

Reply:
ok fixed the types, so now only have outputDir and encryptedWalletPath